shap/explainers/_explainer.py

```diff
@@ -6,7 +6,7 @@
 
 
 class Explainer():
-    def __init__(self, model, masker=None, link=links.identity, algorithm="auto", output_names=None):
+    def __init__(self, model, masker=None, link=links.identity, algorithm="auto", output_names=None, fixed_context = 1):
```

Comment:
it's weird that fixed_context isn't an enum, but not related to this PR.  Any reason it can't be an enum @slundberg ?